CheckDrive.Api/CheckDrive.Application/Extensions/DependencyInjection.cs

```diff
@@ -1,26 +1,43 @@
-﻿using CheckDrive.Application.Interfaces.Review;
+﻿using CheckDrive.Application.Interfaces.Authorization;
+using CheckDrive.Application.Interfaces.Review;
+using CheckDrive.Application.Services.Authorization;
 using CheckDrive.Application.Services.Review;
+using CheckDrive.Domain.Authorization;
+using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 
 namespace CheckDrive.Application.Extensions;
 
 public static class DependencyInjection
 {
-    public static IServiceCollection RegisterApplication(this IServiceCollection services)
+    public static IServiceCollection RegisterApplication(
+        this IServiceCollection services,
+        IConfiguration configuration)
     {
         AddServices(services);
-
+        AddConfigurations(services,configuration);
         services.AddAutoMapper(AppDomain.CurrentDomain.GetAssemblies());
 
         return services;
     }
 
     private static void AddServices(IServiceCollection services)
     {
+        services.AddScoped<JwtHandler>();    
+        services.AddScoped<IAuthService, AuthService>();
+        services.AddScoped<IEmployeeRegistrationService, EmployeeRegistrationService>();
```

Comment:
There is an account service recently added to the API, maybe you should use this service for registration purposes?